### ISSUE TITLE
Instructions for Debian 12 (Bookworm) should be in deb822 format

### ISSUE THREAD

**Issue by 4wk-** [openzfs/openzfs-docs#519]
Starting with Bookworm, it's recommended to write apt sources files in `deb822` format.

Instead of [actual documentation](https://github.com/openzfs/openzfs-docs/blob/master/docs/Getting%20Started/Debian/index.rst):
```debsources
vi /etc/apt/sources.list.d/bookworm-backports.list

deb http://deb.debian.org/debian bookworm-backports main contrib
deb-src http://deb.debian.org/debian bookworm-backports main contrib
```

It should be:
```sourceslist
vi /etc/apt/sources.list.d/bookworm-backports.sources

Types: deb deb-src
URIs: http://deb.debian.org/debian
Suites: bookworm-backports
Components: main contrib
Signed-By: /usr/share/keyrings/debian-archive-keyring.gpg
```

- https://manpages.debian.org/bookworm/apt/sources.list.5.en.html#DEB822-STYLE_FORMAT
- https://repolib.readthedocs.io/en/latest/deb822-format.html (RepoLib is Python-related, but I think it's still interesting)

--
Working as expected:
```
# apt-cache madison zfsutils-linux
zfsutils-linux | 2.2.4-1~bpo12+1 | http://deb.debian.org/debian bookworm-backports/contrib amd64 Packages
```

**Comment by gmelikov** [comment]
While I'm not against it, traditional format is still used and not explicitly described as deprecated.

Fun thing is that even weekly build https://cdimage.debian.org/cdimage/weekly-live-builds/amd64/iso-hybrid/ use traditional format for /etc/apt/sources.list

I propose to leave it as is for now for ease of usage (copy-pasting).

**Comment by 4wk-** [comment]
You're right; even on a fresh "netinstall" Debian image, `/etc/apt/sources.list` file is still written in regular syntax (not `deb822`).
I guess `/etc/apt/sources.list.d/debian.sources` file is packed with [Docker bookworm image](https://github.com/debuerreotype/debuerreotype/pull/142) (a bit "early" :))

**Comment by gmelikov** [comment]
I close it for now, we may get back to this topic when stable Debian release will have it.
